src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
+        int i;
```

Comment:
Variable 'i' should be declared within the for-loop to limit its scope to only where it's needed.